plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/editor/context/project/ProjectContextProvider.kt

```diff
@@ -275,7 +275,7 @@ class ProjectContextProvider(val project: Project, private val encoderServer: En
         var currentTotalFileSize = 0L
         val featureDevSessionContext = FeatureDevSessionContext(project)
         val allFiles = mutableListOf<VirtualFile>()
-        project.guessProjectDir()?.let {
```

Comment:
probably need tests

Reply:
We are making a new LSP release in 1-2 weeks. Tests will be added as part of below PRs

https://github.com/aws/aws-toolkit-jetbrains/pull/4970
https://github.com/aws/aws-toolkit-jetbrains/pull/4971
https://github.com/aws/aws-toolkit-jetbrains/pull/4972
https://github.com/aws/aws-toolkit-jetbrains/pull/4978
https://github.com/aws/aws-toolkit-jetbrains/pull/4976